podcasts.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -42,17 +42,17 @@
         "repositoryURL": "https://github.com/firebase/boringssl-SwiftPM.git",
         "state": {
           "branch": null,
-          "revision": "79db6516894a932d0ddaff3b05b9da1e4f6c4069",
-          "version": "0.9.0"
+          "revision": "dd3eda2b05a3f459fc3073695ad1b28659066eab",
+          "version": "0.9.1"
         }
       },
       {
         "package": "DifferenceKit",
         "repositoryURL": "https://github.com/ra1028/DifferenceKit",
         "state": {
           "branch": null,
-          "revision": "62745d7780deef4a023a792a1f8f763ec7bf9705",
-          "version": "1.2.0"
```

Comment:
I'm not really sure why this has happened, I'm yet to use swiftpm.
Is this normal or have I triggered an update some how 🤔

Reply:
Hmm, you probably triggered an update. Could you please revert this file?